packages/terra-application-utility/src/utility/_UtilityMenu.jsx

```diff
@@ -367,7 +376,7 @@ class UtilityMenu extends React.Component {
           footer={footer}
           fill={isHeightBounded}
           className={menuClassNames}
-          role="navigation"
+          role={menuRole}
```

Comment:
I think this makes more sense to move this role to `UL` element that wraps the menu items. Left more detailed info about this on the GitHub issue: https://github.com/cerner/terra-framework/issues/272#issuecomment-462487936

Reply:
[added role="menu"](https://github.com/cerner/terra-framework/pull/501/commits/36eb776f7973f67fd6cda910594a53aaadcaf71f) to utility-menu-body ul tag as discussed with @JakeLaCombe

Reply:
It looks like aXe tool we use is now throwing warnings about the addition of role="menu". We probably need to dig into this some and figure out another option. I don't have any thoughts on what change is needed currently but will try and post back on this PR when I get some time to dig into this some more.

Reply:
Looks like you got this resolves @supreethmr, nice work!

Reply:
thank you. :)

Reply:
thank you..!